Application/Dtos/Blog/PostDetailsDto.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Application.Dtos.Blog
+{
+    public class PostDetailsDto
+    {
+        public Guid Id { get; set; }
+        public string Message { get; set; }
+        public string Title { get; set; }
+        public List<CommentVm> Comments { get; set; }
```

Comment:
If you decided to separate ViewModels from DTOs, don't mix them. Create a `CommentDto` class and use it here.